update authorization_details (#318)
italia/eidas-it-wallet-docs: docs/en/pid-eaa-issuance.rst

    * - **authorization_details**
      - Array of JSON Objects. Each JSON Object MUST include the following claims:

            - **type**: REQUIRED. It MUST be set to ``openid_credential``,
            - **credential_configuration_id**: REQUIRED when ``format`` parameter is not present. JSON String. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata. For example, in the case of the PID, it MUST be set to ``PersonIdentificationData``. It MUST NOT be present if ``format`` parameter is present.

>             - **format**: REQUIRED when ``credential_configuration_id`` parameter is not present. String identifying the format of the Credential requested by the Wallet. It MUST NOT be present if ``credential_configuration_id`` parameter is present.

Reviewer: This is a breaking change. Current implementation requires only `credential_configuration_id` as a mandatory parameter and `format` parameter is not considered. We will align this in the next milestone.

>             - **format**: REQUIRED when ``credential_configuration_id`` parameter is not present. String identifying the format of the Credential the Wallet needs. This Credential format identifier determines further claims in the authorization details object needed to identify the Credential type in the requested format. It MUST NOT be present if ``credential_configuration_id`` parameter is present.

Reviewer 2: ```suggestion
            - **format**: REQUIRED when ``credential_configuration_id`` parameter is not present. String identifying the format of the Credential requested by the Wallet. It MUST NOT be present if ``credential_configuration_id`` parameter is present.
```
